test/acceptance/features/injectBindingsAsFiles.feature

```diff
@@ -323,3 +323,181 @@ Feature: Bindings get injected as files in application
+            spec:
+                application:
+                    group: apps
+                    version: v1
+                    resource: deployments
+                    name: $scenario_id
+                bindAsFiles: true
+                detectBindingResources: true
+                services:
+                  - group: ""
+                    version: v1
+                    kind: Service
+                    name: $scenario_id-2
+            """
+        Then Service Binding is ready
+        * Service Binding has the binding secret name set in the status
+        * The application "$scenario_id" container has only one env "SERVICE_BINDING_ROOT" set
+
+    Scenario: SERVICE_BIDNING_ROOT is not defined twice in the deployment after binding two services
+        Given Generic test application is running
+        * The env var "SERVICE_BINDING_ROOT" is not available to the application
+        * The Service is present
+            """
+            apiVersion: v1
+            kind: Service
+            metadata:
+                name: $scenario_id-2
+                annotations:
+                    service.binding/service2: path={.metadata.name}
+            spec:
+                selector:
+                    name: $scenario_id
+                ports:
+                  - port: 80
+                    targetPort: 8080
+            """
+        * The Service is present
+            """
+            apiVersion: v1
+            kind: Service
+            metadata:
+                name: $scenario_id-3
+                annotations:
+                    service.binding/service3: path={.metadata.name}
+            spec:
+                selector:
+                    name: $scenario_id
+                ports:
+                  - port: 80
+                    targetPort: 8080
+            """
+        When Service Binding is applied
+            """
+            apiVersion: binding.operators.coreos.com/v1alpha1
+            kind: ServiceBinding
+            metadata:
+                name: $scenario_id
+            spec:
+                application:
+                    group: apps
+                    version: v1
+                    resource: deployments
+                    name: $scenario_id
+                bindAsFiles: true
+                detectBindingResources: true
+                services:
+                  - group: ""
+                    version: v1
+                    kind: Service
+                    name: $scenario_id-2
+                  - group: ""
+                    version: v1
+                    kind: Service
+                    name: $scenario_id-3
+            """
+        Then Service Binding is ready
+        * Service Binding has the binding secret name set in the status
+        * The application "$scenario_id" container has only one env "SERVICE_BINDING_ROOT" set
+
+    Scenario: SERVICE_BIDNING_ROOT is not defined twice in the deployment after binding to service without bindable data
```

Comment:
as talked about this, no binding data should fail binding process, and that should be solved separatelly

Reply:
+1 https://github.com/redhat-developer/service-binding-operator/pull/1031